Models/wardrobe.py

```diff
@@ -7,4 +9,4 @@ class Wardrobe(Base):
 
     id: Mapped[int] = mapped_column(primary_key=True)
     user_id: Mapped[int] = mapped_column(ForeignKey("User.id"))
```

Comment:
There is still problem with relations, such as:
   raise sa_exc.AmbiguousForeignKeysError(
sqlalchemy.exc.AmbiguousForeignKeysError: Could not determine join condition between parent/child tables on relationship Collection.user - there are multiple foreign key paths linking the tables.  Specify the 'foreign_keys' argument, providing a list of those columns which should be counted as containing a foreign key reference to the parent table.

I will try to resolve it in few days, for now I will add it, but it will be commented